### PR TITLE
Add development tips for Bash on Windows

### PR DESCRIPTION
**Pull request by nolanlawson**
I managed to get Mastodon running on Bash on Ubuntu on Windows, but it took some small extra steps.

### CODE REVIEW

File: Running-Mastodon/Development-guide.md [hunk 1]
```diff
@@ -128,6 +128,23 @@ Go to http://localhost:3000 to see your development instance.
 
 Admin account is `admin@localhost:3000`. Password is `mastodonadmin`.
 
+## Bash on Windows
+
+To run Mastodon on [Bash on Ubuntu on Windows](https://msdn.microsoft.com/en-us/commandline/wsl/about), you can follow the Ubuntu instructions with some small tweaks.
```
Comment (wxcafe) [inline Running-Mastodon/Development-guide.md]: I think we could okay this if it's specified that running Mastodon on bash on windows is TOTALLY UNSUPPORTED AND NOT RECOMMENDED.

Comment (nolanlawson) [inline Running-Mastodon/Development-guide.md]: Isn't it kind of implied given this is in the Development guide and not the Production guide? Also most of the marketing around BoW promotes it purely as a developer tool.

Comment (wxcafe) [inline Running-Mastodon/Development-guide.md]: yeah totally, I meant even for development. Like, that we don't support users who have trouble with development on bash on windows.

Comment (nolanlawson) [inline Running-Mastodon/Development-guide.md]: Ok, updated.

**Comment by nolanlawson** [comment]
I just re-tested on Bash on Ubuntu on Windows, and it looks like once the Fall Creators Update ships (early October), none of these extra steps are required. :smiley: You can just follow the Ubuntu/Linux guide and everything works, including rbenv and ffmpeg. So we can probably close this PR!

**Comment by wxcafe** [comment]
that's pretty cool! I'm sorry that it didn't get merged before now tho :<

**Comment by nolanlawson** [comment]
No prob! :)

**Pull request closed without merging**